Src/BootCamp.Chapter/PeopleAndBalanceManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace BootCamp.Chapter
+{
+    public class PeopleAndBalanceManager
+    {
+        private PersonAndBalance[] _peopleAndBalances;
+        private AnswerFormatter _answerFormatter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_answerFormatter' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)